sklearnex mention in auto-sklearn docs pre-review (#1)
Alexsandruss/auto-sklearn: README.md

predictions = cls.predict(X_test)
```

## auto-sklearn acceleration with sklearnex

You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is extension that speedups sklearn algorithms through patching with highly optimized machine learning kernels.

Install sklearnex with pip or conda:
```bash
pip install scikit-learn-intelex
conda install scikit-learn-intelex
conda install -c conda-forge scikit-learn-intelex
```

> Insert these two lines of code before auto-sklearn and sklearn imports to get acceleration:

Reviewer: My suggestion is to add information about unpatch_sklearn() too.

Author: Added

predictions = cls.predict(X_test)
```

## auto-sklearn acceleration with sklearnex

> You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is extension that speedups sklearn algorithms through patching with highly optimized machine learning kernels.

Reviewer 2: multiple suggestions with the main mistakes corrected:

```suggestion
You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is an extension that speeds up scikit-learn algorithms through patching with highly optimized machine learning kernels.
```

it's also a good idea to use the full name of the extension:

```suggestion
You can accelerate auto-sklearn with [Intel(R) Extension for Scikit-Learn (sklearnex)](https://github.com/intel/scikit-learn-intelex). It is an extension that speeds up scikit-learn algorithms through patching with highly optimized machine learning kernels.
```

I have doubts about the `through patching with highly optimized machine learning kernels` phrase, I don't think it's clear enough for the reader, so here is a way to rephrase this:

```suggestion
You can accelerate auto-sklearn with [Intel(R) Extension for Scikit-Learn (sklearnex)](https://github.com/intel/scikit-learn-intelex). The acceleration is achieved through **patching**: replacing scikit-learn algorithms with their optimized versions provided by the extension.
```

you may also refer users to our docs where patching is explained: https://intel.github.io/scikit-learn-intelex/what-is-patching.html

> Install sklearnex with pip or conda:

Reviewer 2: I think it makes sense to refer to detailed install instructions/supported configurations

Author: System reqs are referred now

```bash
pip install scikit-learn-intelex
conda install scikit-learn-intelex
conda install -c conda-forge scikit-learn-intelex
```

> Insert these two lines of patching code before auto-sklearn and sklearn imports to get acceleration:

Reviewer 2: I suggest this version:
```suggestion
To accelerate auto-sklearn, patch scikit-learn before importing auto-sklearn and sklearn:
```

this is also ok:
```suggestion
To accelerate auto-sklearn, insert the following two lines of patching code before auto-sklearn and sklearn imports:
```

Author: Changed to 2nd suggestion

> conda install -c conda-forge scikit-learn-intelex

Reviewer 3: do we need to have conda-forge line here?

Author: conda-forge is mostly popular as main channel, so mention about sklearnex availability at conda-forge is useful for many users who don't like to complicate environment with packages from 2 and more channels

Reviewer 3: It's ok to mention, but i don't think it worth adding command example as it's add complexity - let's pick single line that we will give users - for now we are putting lot of details in to readme. 

@outoftardis  - can we insert copy button in readme?   i.e. instead of select - right click - copy click  we will use just copy button

Reviewer 2: @napetrov it's already there :) in the right upper corner

![image](https://user-images.githubusercontent.com/17865025/143430989-2aeca8d9-9180-4e4a-a666-8a6a872a14c6.png)

Reviewer 3: then we for sure need single line not multiple lines in block

Author: Lines are splitted now

Author: oh, forgot that it is about .md, I leaved one line at README, .rst format at web site lacks copy button, but I think separate lines look better

```python
from sklearnex import patch_sklearn
patch_sklearn()

import autosklearn.classification
```

> Call unpatching and reimport auto-sklearn and sklearn to return original implementation:

Reviewer 2: ```suggestion
To return to the original scikit-learn implementation, unpatch scikit-learn and reimport auto-sklearn and sklearn:
```

Author: Changed to suggestion

> To return to the original scikit-learn implementation, unpatch scikit-learn and reimport auto-sklearn and sklearn:

Reviewer 3: 1. Do we have any sort of agreement that we also can be added to readme? 
2. I think we need details on unpatching here - just link to our github/docs would be sufficient to close details.

Author: 1. No, only general agreement
2. There is only few lines about unpatching in sklearnex docs and their rephrase is here in this PR

Reviewer 3: They have 1 screen read me now and you would like to add another screen there..... 

I would doubt that they would accept such addition there.  Realistically i think we can hope for single lien there. 

our acceleration is not core feature for them, even not close..

Author: I leaved one line in README with link to details in auto-sklearn docs
